e2e/cypress/integration/messaging/mention_autocomplete_overlap_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and go to /
```

Comment:
I think we should add `cy.viewport(1280, 900)` since with the current viewport settings the test doesn't pass.

Reply:
Thanks for the comment! Maybe I am mistaken but... is not that the point? The test should fail if something is not working correctly, and it appears like things are not working as expected. Maybe the reason behind the test is a different one, and I am not grasping the reason to do it, but it feels that if we force the test passing that way, the test will make no sense.

Reply:
Not mentioned in the ticket but I'm fine with @hahmadia's suggestion to test on larger viewport height and another test with your point (@larkox) on smaller height which I filed a ticket to clarify/fix expectation - https://mattermost.atlassian.net/browse/MM-20310. Let's wait for that to resolve then.

Reply:
This and other requested changes added on 8b9a36b . I will wait for this to be resolved, in case we have to make another change.

Reply:
Test on default viewport is failing due to the said https://mattermost.atlassian.net/browse/MM-20310.  Again, let's wait for its fix which is scheduled for 5.18.

Reply:
fyi, awaiting PR https://github.com/mattermost/mattermost-webapp/pull/4360

Reply:
@saturninoabril Seems it's been merged now. Could we get a re-review? Thanks!

Reply:
Sorry for the delay.  Done review, requested for a change.